### PR TITLE
Allows deleting subjects

### PR DESCRIPTION
**Pull request by GiltosMx**
Are we hiding in the shadows?
Forget the pain, and forget the sorrows.

Adds a button on the ClassInfoDialog that deletes the selected Subject
from the database.

Still need to update all of the other components after doing the subject deletion,
since it deletes also all of the tasks and times, so we shouldn't show those things
on the other tabs after closing the dialog.

### CODE REVIEW

File: src/controllers/EditSubjectDaysController.java [hunk 1]
```diff
@@ -176,7 +176,7 @@ public String getChoiceBoxDay(int choiceBoxIndex){
 			day = "SUNDAY";
 			break;
 		default: 
-			day = "N/A";
+			day = "MONDAY";
```
Comment (GalaxiasKyklos) [inline src/controllers/EditSubjectDaysController.java]: Thank you!!

File: src/controllers/ManageSubjectsController.java [hunk 2]
```diff
@@ -163,29 +163,35 @@ public void btnCancelAction() {
...
+        GridPane pane = loader.load();
+        ((Refresher)loader.getController()).setParent(self);
+
+        pane.setStyle(Main.getThemeString());
+        dialogStage.setScene(new Scene(pane));
+        dialogStage.show();
+
+        // Sets the textArea default value
+        TextArea paneTextArea = (TextArea) pane.getChildren().get(4);
+        paneTextArea.setText(classes.get(0).toString
```
Comment (Rudycc) [inline src/controllers/ManageSubjectsController.java]: I believe theres a missing parenthesis here, as well as ;

File: src/controllers/ClassInfoController.java [hunk 3]
```diff
@@ -42,6 +47,26 @@ public void initialize(URL location, ResourceBundle resources) {
 		this.classes = SubjectDatabaseController.getAllClasses();
 		this.classIndex = 0;
 	}
+  
+	public void btnDeleteSubjectAction(){
+		if(classes.size() > 0)
+			SubjectDatabaseController.deleteSubject(classes.get(classIndex).getSubjectId());
+
```
Comment (Rudycc) [inline src/controllers/ClassInfoController.java]: You should call the parent.refreshData() method to let know the parent that the info has changed

Comment (GiltosMx) [reply src/controllers/ClassInfoController.java]: Roger roger, all done :robot:

**Pull request merged**